app/models/group.rb

```diff
@@ -560,7 +560,7 @@ def self.refresh_automatic_group!(name)
       when :staff
         "SELECT id FROM users WHERE id <= 0 OR (NOT admin AND NOT moderator) OR staged"
       when :trust_level_0, :trust_level_1, :trust_level_2, :trust_level_3, :trust_level_4
-        "SELECT id FROM users WHERE id <= 0 OR trust_level < #{id - 10} OR staged"
+        "SELECT id FROM users WHERE id < -1 OR trust_level < #{id - 10} OR staged"
```

Comment:
Exclude the system user when booting users from TL groups.

Reply:
This change also needs to be made to the staff/admins/moderators groups too, because system user is admin and moderator:

https://github.com/discourse/discourse/blob/a367ede57461671a72cf431b1be2bc51d684bcce/db/fixtures/009_users.rb#L25-L26